ext/smoke/packages/steps/setup-master.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
```

Comment:
This should probably take the collection as a param. Johnson's going to be using puppet6.